app/jobs/send_daily_triage_email_job.rb

```diff
@@ -28,4 +28,14 @@ def skip_daily_email?(user)
   def email_decider(user)
     @email_decider ||= EmailDecider.new(user.days_since_last_clicked, minimum_frequency: user.email_frequency)
   end
+
+  DEFAULT_EMAIL_TIME_OF_DAY = Time.utc(2000, 1, 1, 17, 00, 0)
+
+  def before_email_time_of_day?(user)
```

Comment:
Done. 

I thought this might not work in the case that a different date ever got stored by accident. e.g. via console or database manipulation. 

But this can't happen. The 2000-1-1 date is never stored in the table. It gets added when it's turned into a ruby object.

```
\d users
...
| email_time_of_day      | time without time zone      |                                                                  |
...
```

```
triage_development> select email_time_of_day from users;
+---------------------+
| email_time_of_day   |
|---------------------|
| 15:31:01.986967     |
+---------------------+
```